reyna/src/com/b2msolutions/reyna/services/StoreService.java

```diff
@@ -38,28 +38,54 @@ public static void setCellularDataBlackout(Context context, TimeRange range) {
         new Preferences(context).saveCellularDataBlackout(range);
     }
 
-	@Override
-	protected void onHandleIntent(Intent intent) {
+    public static void setStorageSizeLimit(Context context, long limit) {
+        if (limit <= 0) {
+            return;
+        }
+
+        Preferences preferences = new Preferences(context);
+        preferences.saveStorageSize(limit);
+    }
+
+    public static long getStorageSizeLimit(Context context) {
+        Preferences preferences = new Preferences(context);
+        return preferences.getStorageSize();
+    }
+
+    public static void resetStorageSizeLimit(Context context) {
+        Preferences preferences = new Preferences(context);
+        preferences.saveStorageSize(-1);
```

Comment:
you might need preferences clear method

preferences clear should do

  SharedPreferences.Editor editor = getEditor(context);
   editor.remove(key);
   editor.commit();

@Youhana-Hana